src/core/reference.js

```diff
@@ -40,14 +56,22 @@
  */
 
 /**
+ * The if-else statement helps control the flow of your code.
+ * A condition is placed between the parenthesis following 'if'.
+ * When that condition evalues to <a href="https://developer.mozilla.org/en-US/docs/Glossary/truthy">truthy</a>,
+ * the code between the preceding curly braces is run.
+ * Alternatively, when the condition evaluates to <a href="https://developer.mozilla.org/en-US/docs/Glossary/Falsy">falsy</a>,
+ * the code between the curly braces that precedes 'else' is run instead.
```

Comment:
> the curly braces that precedes 'else'

probably should have been "the curly braces that comes after 'else'"? As precede means comes before.

Reply:
Also, "Haha Thanks!" 🤦‍♀

Reply:
or "the curly braces that follow"